src/zeek.bif

```diff
@@ -1330,61 +1331,71 @@ function all_set%(v: any%) : bool
 
 %%{
 static zeek::Func* sort_function_comp = nullptr;
-static std::vector<const zeek::ValPtr*> index_map;	// used for indirect sorting to support order()
 
-bool sort_function(const zeek::ValPtr& a, const zeek::ValPtr& b)
+// Used for indirect sorting to support order().
+static std::vector<zeek::ZAMValUnion> index_map;
+
+// Yuck, we need a variable to hold the type.  Luckily, this won't be
+// invoked concurrently for two different types.
+static zeek::IntrusivePtr<zeek::Type> sort_type;
+static bool sort_type_is_managed = false;
+
+bool sort_function(zeek::ZAMValUnion& a, zeek::ZAMValUnion& b)
 	{
-	// Sort missing values as "high".
-	if ( ! a )
-		return 0;
-	if ( ! b )
-		return 1;
+	// Unable to fully do this:
+	//	Sort missing values as "high".
+	// But we can for managed types.
+
+	if ( sort_type_is_managed )
+		{
+		if ( ! a.managed_val )
+			return 0;
```

Comment:
```suggestion
			return false;
```

Reply:
Certainly a reasonable change, but note that that's not code I've touched, it's like that in master.

Reply:
Weird. I thought I had fixed all of those a while back. I must have missed these two.